### PR TITLE
feat(crowdnode): UI enhancements

### PR DESCRIPTION
**Pull request by tikhop**
## Issue being fixed or feature implemented

UI enhancements 

## What was done?

- [x] Align colors with the rest of UI
- [x] Dark mode fixes 
- [x] Minor UI improvements

## How Has This Been Tested?

QA, Manually

## Breaking Changes

N/A

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountDetailsController.swift [hunk 1]
```diff
@@ -42,6 +42,8 @@ final class OnlineAccountDetailsController: UIViewController {
     }
 
     private func configureHierarchy() {
+        view.backgroundColor = .dw_secondaryBackground()
```
Comment (Syn-McJ) [inline DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountDetailsController.swift:45]: This creates some issues with UI. By design, we have a white background there with gray address boxes: https://www.figma.com/file/84tUBQA9DYU1svyQx8tnbk/Dash-Wallet%E3%83%BBIn-process%E3%83%BBAndroid?node-id=10643%3A18770&t=z4LVy8U1iXqA1XMw-0

File: DashWallet/Sources/UI/CrowdNode/New Account/NewAccountViewController.swift [hunk 2]
```diff
@@ -80,11 +80,7 @@ final class NewAccountViewController: UIViewController, UITextViewDelegate {
 extension NewAccountViewController {
     private func configureHierarchy() {
         definesPresentationContext = true
-        view.backgroundColor = UIColor.dw_background()
-        let appearance = UINavigationBarAppearance()
-        appearance.configureWithTransparentBackground()
-        navigationItem.standardAppearance = appearance
-        navigationItem.scrollEdgeAppearance = appearance
+        view.backgroundColor = .dw_secondaryBackground()
```
Comment (Syn-McJ) [inline DashWallet/Sources/UI/CrowdNode/New Account/NewAccountViewController.swift:83]: This creates issues with UI. By design, we have a white background there with a gray address box: https://www.figma.com/file/aghEG2PAqmXghOWi5BoHg5/Dash-Wallet%E3%83%BBApproved%E3%83%BBAndroid?node-id=2145%3A31181&t=FLesXM2Oin3xbgCB-0

File: DashWallet/Sources/UI/CrowdNode/New Account/AccountCreatingController.swift [hunk 3]
```diff
@@ -53,10 +53,7 @@ final class AccountCreatingController: UIViewController {
 
 extension AccountCreatingController {
     private func configureHierarchy() {
-        let appearance = UINavigationBarAppearance()
-        appearance.configureWithTransparentBackground()
-        navigationItem.standardAppearance = appearance
-        navigationItem.scrollEdgeAppearance = appearance
+        view.backgroundColor = .dw_secondaryBackground()
```
Comment (Syn-McJ) [inline DashWallet/Sources/UI/CrowdNode/New Account/AccountCreatingController.swift:56]: This creates problems with UI. By design, we have a white background with a gray button here: https://www.figma.com/file/84tUBQA9DYU1svyQx8tnbk/Dash-Wallet%E3%83%BBIn-process%E3%83%BBAndroid?node-id=13128%3A14702&t=z4LVy8U1iXqA1XMw-0

File: DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountConfirmationController.swift [hunk 4]
```diff
@@ -85,6 +85,8 @@ final class OnlineAccountConfirmationController: UIViewController {
 
 extension OnlineAccountConfirmationController {
     private func configureHierarchy() {
+        view.backgroundColor = .dw_secondaryBackground()
```
Comment (Syn-McJ) [inline DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountConfirmationController.swift:88]: By design, we have a white background with gray address boxes here: https://www.figma.com/file/aghEG2PAqmXghOWi5BoHg5/Dash-Wallet%E3%83%BBApproved%E3%83%BBAndroid?node-id=2159%3A31468&t=FLesXM2Oin3xbgCB-0

File: DashWallet/Sources/UI/CrowdNode/Portal/Views/CrowdNodeCell.swift [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+                verifyButton.isHidden = true
+            }
+        }
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        contentView.backgroundColor = .dw_background()
+    }
```
Comment (Syn-McJ) [inline DashWallet/Sources/UI/CrowdNode/Portal/Views/CrowdNodeCell.swift:92]: This seems to be breaking the highlight effect on pressing the element. Instead, we have a flickering border now which looks like a bug.

**Review by Syn-McJ — CHANGES_REQUESTED** [review]
There are some issues that need to be fixed.

**Review by Syn-McJ — APPROVED** [review]
Looks good

File: DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountDetailsController.swift (hunk 1 above)
Comment (tikhop) [reply DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountDetailsController.swift:45]: Fixed: Ended up introducing new color for blocks like 'Primary Dash address'

File: DashWallet/Sources/UI/CrowdNode/New Account/NewAccountViewController.swift (hunk 2 above)
Comment (tikhop) [reply DashWallet/Sources/UI/CrowdNode/New Account/NewAccountViewController.swift:83]: Fixed: Ended up introducing new color for blocks like 'Primary Dash address'

File: DashWallet/Sources/UI/CrowdNode/New Account/AccountCreatingController.swift (hunk 3 above)
Comment (tikhop) [reply DashWallet/Sources/UI/CrowdNode/New Account/AccountCreatingController.swift:56]: Fixed: Used the same we use for blocks like 'Primary Dash address' for the button

File: DashWallet/Sources/UI/CrowdNode/Portal/Views/CrowdNodeCell.swift (hunk 5 above)
Comment (tikhop) [reply DashWallet/Sources/UI/CrowdNode/Portal/Views/CrowdNodeCell.swift]: Fixed: Now we set a background color for the cell to make sure selection works as expected

File: DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountConfirmationController.swift (hunk 4 above)
Comment (tikhop) [reply DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountConfirmationController.swift:88]: Fixed: Use 'QuaternaryFillColor' color for blocks like address

**Review by Syn-McJ — APPROVED** [review]

**Review by Syn-McJ — APPROVED** [review]
Thre were minor conflicts, but they are resolved. We can merge this now.

**Pull request merged**